docs/install/pypi.md

```diff
@@ -0,0 +1,135 @@
+# Install from PyPI
+
+Install OFRAK quickly using pip for the simplest setup.
+
+## Prerequisites
+
+- Python 3.9+ and pip
+
+## Quick Install
+
+```bash
+pip install ofrak
+```
+
+This installs the core `ofrak` package and `ofrak_patch_maker` along with all Python dependencies.
+
+## Verify Installation
+
+
+
+```mermaid
+
+```
+
+```bash
+ofrak list
+```
+
+This lists all installed OFRAK modules and components.
+
+## What's Included
+
+The PyPI installation provides:
+-  OFRAK core functionality
+-  Patch maker
+- L Disassembler backends (require separate installation)
+- L Some analysis components (optional)
+
+## Installing Disassemblers
+
+### angr + Capstone (Recommended for beginners)
+
+```bash
+pip install ofrak_angr ofrak_capstone
+```
+
+These work together: angr identifies functions, capstone disassembles code.
+
+### Ghidra
+
+```bash
+pip install ofrak_ghidra
+```
+
+Requires Ghidra installed separately. See [Ghidra Backend Guide](../user-guide/disassembler-backends/ghidra.md).
+
+### Binary Ninja
+
+```bash
+pip install ofrak_binary_ninja
+```
+
+Requires Binary Ninja license. See [Binary Ninja Backend Guide](../user-guide/disassembler-backends/binary_ninja.md).
+
+## Handling Missing Dependencies
+
+OFRAK integrates many external tools. Not all can be installed via pip.
+
+### Option 1: Exclude Missing Dependencies
+
+Use the `-x` flag to skip components with missing dependencies:
+
+```bash
+ofrak unpack -x <file>
+```
+
+Or in Python:
+
+```python
+from ofrak import OFRAK
+
+ofrak = OFRAK(
+    exclude_components_missing_dependencies=True
+)
+```
+
+**Note**: This prevents errors but means OFRAK can't use those components. For example, without `pigz`, gzip files can't be processed.
```

Comment:
While the spirit of what you're saying is true, this might be a bad example, because if I remember correctly, OFRAK actually falls back on `gzip` when `pigz` is not present...

Reply:
Updated to use APK example.